sdk-docs/how-to/flask.md

```diff
@@ -0,0 +1,103 @@
+# Add an environment variable
+
+A charm configuration can be added if you require environment variables, for
+example, to pass a token for a service. For example, add the following
+configuration in `charmcraft.yaml`:
+
+```yaml
+config:
+  options:
+    token:
+      description: The token for the service.
+      type: string
+      required: true
+```
+
+PaaS App Charmer will map this as the `FLASK_TOKEN` environment variable. In
+general, the environment variable name is `FLASK_<config option name>` where the
+config option name will be converted to upper case. The configuration can be set
+on the deployed charm using `juju config <app name> token=<token>`
+
+# Database migration
+
+If your app depends on a database it is common to run a database migration
+script before app startup which, for example, creates or modifies tables. This
+can be done by including the `migrate.sh` script in the root of your project. It
+will be executed with the same environment variables and context as the flask
+app.
+
+If the migration script fails, the app won't be started and the app charm will
+go into blocked state. The migration script will be run on every unit and it is
+assumed that it is idempotent (can be run multiple times) and that it can be run
+on multiple units at the same time without causing issues. This can be achieved
+by, for example, locking any tables during the migration.
+
+# Including extra files in the OCI image
+
+The following files are included in the image by default from the root of the project:
+
+- `app`
+- `app.py`
+- `migrate`
+- `migrate.sh`
+- `migrate.py`
+- `static`
+- `templates`
+
+To change this, the following snippet needs to be added to the `rockfile.yaml`:
```

Comment:
This is interesting because the rockcraft docs explicitly use the include / exclude syntax (where to exclude you have to use -), and if all the files mentioned above where actually included we'd have to exclude them explicitly, but the code below suggests that there isn't really a "default" -- rockcraft just picks up whatever it sees in the prime key, and  the prime key by default lists all of the above. (Might seem like a nitpick but it confused me for a bit.)

Reply:
Do you have any suggestion on how this could be clearer please?